README.md

```diff
@@ -68,4 +79,20 @@ Tests are implemented using [mocha](https://mochajs.org/). They can be run using
 npm test
 ```
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+[(back to top)](#bsi-secvisogram-csaf-validator-service)
+
+## Contributing
+
+You can find our guidelines here [CONTRIBUTING.md](https://github.com/secvisogram/secvisogram/blob/main/CONTRIBUTING.md)
```

Comment:
Same as https://github.com/secvisogram/csaf-validator-lib/pull/35#discussion_r934409424

Reply:
Done in https://github.com/secvisogram/secvisogram/pull/292/commits/51f4151e394dc9e394d4097a8be594c6c1f3f43d